workspaces/src/types/account.rs

```diff
@@ -147,6 +147,11 @@ impl Account {
 
         Ok(())
     }
+
+    /// Get the keys of this account. The public key can be retrieved from the secret key.
+    pub fn keys(&self) -> SecretKey {
+        SecretKey(self.signer.0.secret_key.clone())
+    }
```

Comment:
I think it is fine to only return `SecretKey`, but I would argue that `keys` is not a good name in this case because of plurality. Maybe just `key` or `secret_key` while leaving the clarification that public key can be retrieved from the secret key?

Reply:
Also, noting it's probably best if we keep this representation internally, and then this method can return `&SecretKey`. Cloning implicitly feels weird

Reply:
Gotcha, I'll re-work how `InMemorySigner` is represented internally but shouldn't be too big of a deal to expose it as a `&SecretKey`

Reply:
alright should be good. Ready to be re-reviewed whenever you guys get the chance